chitchat/src/state.rs

```diff
@@ -165,43 +187,43 @@ impl ClusterState {
         // Remove nodes to reset.
         self.node_states
             .retain(|chitchat_id, _| !delta.nodes_to_reset.contains(chitchat_id));
-        // And apply delta.
+
+        // Apply delta.
         for (chitchat_id, node_delta) in delta.node_deltas {
-            let mut node_state_map = self
-                .node_states
-                .entry(chitchat_id)
-                .or_insert_with(NodeState::default);
+            let mut node_state = self.node_states.entry(chitchat_id).or_default();
+
+            if node_state.heartbeat < node_delta.heartbeat {
+                node_state.heartbeat = node_delta.heartbeat;
+                node_state.last_heartbeat = Instant::now();
```

Comment:
I'm wondering why we need `last_heartbeat`. When looking at the code, it seems useless.